src/Uno.UI/UI/Xaml/Controls/CalendarDatePicker/CalendarDatePicker.Android.cs

```diff
@@ -0,0 +1,205 @@
+
+		#endregion
+
+		#region PlaceholderText
+
+
+		public string PlaceholderText
+		{
+			get { return (string)this.GetValue(PlaceholderTextProperty); }
+			set { this.SetValue(PlaceholderTextProperty, value); }
+		}
+
+		public static global::Windows.UI.Xaml.DependencyProperty PlaceholderTextProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"PlaceholderText", typeof(string),
+			typeof(global::Windows.UI.Xaml.Controls.CalendarDatePicker),
+			new FrameworkPropertyMetadata(default(string)));
+
+		#endregion
+
+
+		#region DateFormat
+		public string DateFormat
+		{
+			get { return (string)this.GetValue(DateFormatProperty); }
+			set { this.SetValue(DateFormatProperty, value); }
+		}
+
+		public static global::Windows.UI.Xaml.DependencyProperty DateFormatProperty { get; } =
+		Windows.UI.Xaml.DependencyProperty.Register(
+			"DateFormat", typeof(string),
+			typeof(global::Windows.UI.Xaml.Controls.CalendarDatePicker),
+			new FrameworkPropertyMetadata(default(string)));
+
+
+		#endregion
+
+		#region MaxDateProperty
+		public DateTimeOffset MaxDate
+		{
+			get { return (DateTimeOffset)this.GetValue(MaxDateProperty); }
+			set { this.SetValue(MaxDateProperty, value); }
+		}
+
+		public static DependencyProperty MaxDateProperty { get; } =
+			DependencyProperty.Register("MaxDate", typeof(DateTimeOffset), typeof(CalendarDatePicker),
+			new PropertyMetadata(DateTimeOffset.MaxValue));
+
+		#endregion
+
+		#region MinDate
+		public DateTimeOffset MinDate
+		{
+			get { return (DateTimeOffset)this.GetValue(MinDateProperty); }
+			set { this.SetValue(MinDateProperty, value); }
+		}
+
+		public static DependencyProperty MinDateProperty { get; } =
+			DependencyProperty.Register("MinDate", typeof(DateTimeOffset), typeof(CalendarDatePicker),
+			new PropertyMetadata(DateTimeOffset.MinValue));
+
+		#endregion
+
+#if XAMARIN
+
+		#region Template parts
+		public const string FlyoutButtonPartName = "FlyoutButton";
+		#endregion
+
+		private Button _flyoutButton;
+
+		private bool _isLoaded;
+		private bool _isViewReady;
+
+		protected override void OnApplyTemplate()
+		{
+			base.OnApplyTemplate();
+
+			_flyoutButton = this.GetTemplateChild(FlyoutButtonPartName) as Button;
+
+			var flyoutContent = _flyoutButton?.Content as IFrameworkElement;
+
+			_isViewReady = true;
+
+			SetupFlyoutButton();
+
+			OnApplyTemplatePartial();
+
+		}
+
+		partial void OnApplyTemplatePartial();
+
+		protected override void OnLoaded()
+		{
+			base.OnLoaded();
+
+			_isLoaded = true;
+
+			var flyoutContent = _flyoutButton?.Content as IFrameworkElement;
+
+			if (flyoutContent != null)
+			{
+				SetupFlyoutButton();
+			}
+		}
+
+		private void SetupFlyoutButton()
+		{
+			if (!_isViewReady || !_isLoaded)
+			{
+				return;
+			}
+
+			if (_flyoutButton != null)
+			{
+
+				_flyoutButton.Flyout = new CalendarDatePickerFlyout()
+				{
+					MinDate = MinDate,
+					MaxDate = MaxDate,
+					Date = Date // this.Date.GetValueOrDefault(DateTimeOffset.Now)
```

Comment:
Is the comment needed ?

Reply:
Of course not, it is left-over from tests...